drata/README.md

```diff
@@ -0,0 +1,33 @@
+# Drata
+
+## Overview
+
+Drata is a security and compliance automation platform that continuously monitors and collects evidence of a company's security controls, while streamlining compliance workflows end-to-end to ensure audit readiness.
+
+This integration allows [Drata][5] customers to forward their compliance-related logs and events from Datadog to Drata through an API integration.
+
+## Setup
+
+To set up this integration, you must have an active [Drata account][6]. You must also have proper [admin permissions][7] in Datadog.
+
+### Installation
+
+1. To install this integration, you need to create an API and App key. 
+2. Navigate to your organization settings to [create an API key][2] in Datadog. Give the key a meaningful name such as `Drata`.
+3. Copy and save your API `Key`.
+4. Within your organization settings, [create an application key][3]. 
+5. Copy and save your application key.
```

Comment:
What does the customer do after creating their api and app key? Can you provide more insight here? Thanks!

Reply:
@emarsha94 updated these instructions, and added logos. My team would rather keep the marketing pages, as the dashboards aren't very useful for explaining the integration. If it's easier, I can just remove the images and having nothing on the carousel.